go/pkg/client/cas.go

```diff
@@ -25,11 +25,17 @@ import (
 	log "github.com/golang/glog"
 )
 
+// UploadIfMissingStats reports the CAS hits/misses.
+type UploadIfMissingStats struct {
+	Hits   map[digest.Digest]struct{}
```

Comment:
I think it is fine to return only Hits or Misses and let user calculate opposite

Reply:
Done.